charts/splunk-connect-for-snmp/templates/_helpers.tpl

```diff
@@ -14,6 +14,85 @@
 {{- printf "redis://%s-redis-headless:6379/1" .Release.Name }}
 {{- end }}
 
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "splunk-connect-for-snmp.fullname" -}}
+{{- if .Values.worker.fullnameOverride }}
+{{- .Values.worker.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default (printf "%s-%s" .Chart.Name "user") .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "splunk-connect-for-snmp.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "splunk-connect-for-snmp.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "splunk-connect-for-snmp.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "splunk-connect-for-snmp.labels" -}}
+helm.sh/chart: {{ include "splunk-connect-for-snmp.chart" . }}
+{{ include "splunk-connect-for-snmp.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "splunk-connect-for-snmp.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create }}
+{{- default (include "splunk-connect-for-snmp.fullname" .) .Values.serviceAccount.name }}
+{{- else }}
+{{- default "default" .Values.serviceAccount.name }}
+{{- end }}
+{{- end }}
+
 {{- define "splunk-connect-for-snmp.name" -}}
 {{- default (printf "%s" .Chart.Name ) .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
+
+
+{{/*
+Whether enable traps
+*/}}
+{{- define "splunk-connect-for-snmp.traps.enable" -}}
+{{- if or (and (eq .Values.traps.service.type "LoadBalancer") .Values.traps.loadBalancerIP ) (and (eq .Values.traps.service.type "NodePort") .Values.traps.service.nodePort) }}
+{{- printf "true" }}
+{{- else }}
+{{- printf "false" }}
+{{- end -}}
+{{- end }}
+
+{{/*
+Whether enable polling
+*/}}
+{{- define "splunk-connect-for-snmp.polling.enable" -}}
+{{- if .Values.poller.inventory }}
+{{- printf "true" }}
```

Comment:
Yes, the csv values is required to use polling, so I think `eanbled` switch would be redundant. Also I think it would confuse the users who already use SC4SNMP.